app/src/main/java/com/jienihou/outfitxchange/productdetails/ProductDetailsScreen.kt

```diff
@@ -0,0 +1,78 @@
+package com.jienihou.outfitxchange.productdetails
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
+import androidx.fragment.app.Fragment
+import androidx.navigation.Navigation
+import com.ittianyu.bottomnavigationviewex.BottomNavigationViewEx
+import com.jienihou.outfitxchange.R
+
+/**
+ * ProductDetailsScreen shows the product details of an item
+ */
+class ProductDetailsScreen : Fragment() {
+    private lateinit var bottomNavigationView: BottomNavigationViewEx
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val view = inflater.inflate(R.layout.fragment_product_details_screen, container, false)
+
+        // Logo bar view
+        val toolbar = view?.findViewById(R.id.toolbar) as Toolbar
+        (activity as AppCompatActivity).setSupportActionBar(toolbar)
+        val heartImage = view.findViewById(R.id.outline_heart) as ImageView
+        heartImage.setOnClickListener{
+            if(it.isActivated){
+                Toast.makeText(this.context, "Removed from Favourites", Toast.LENGTH_SHORT).show()
+            } else{
+                Toast.makeText(this.context, "Added to Favourites", Toast.LENGTH_SHORT).show()
+            }
+            it.isActivated = !it.isActivated
+        }
+
+        // Product Detail Page View
+        val image = view.findViewById<ImageView>(R.id.image)
+        val brandName = view.findViewById<TextView>(R.id.product_title)
+        val productStyle = view.findViewById<TextView>(R.id.product_style)
+        val productSize = view.findViewById<TextView>(R.id.product_size)
+        val rentPrice = view.findViewById<TextView>(R.id.rent_price)
+
+        // TODO: Replace with different image if the product image is not found
+        val imageName = arguments?.getInt("imageName") ?: R.drawable.no_image
```

Comment:
This is great, but maybe it would be cleaner to pack all the arguments into a single object and retrieve that instead? Might need to use a [parcelable](https://www.vogella.com/tutorials/AndroidParcelable/article.html) (with kotlin data classes its [fairly trivial to use](https://stackoverflow.com/questions/33551972/is-there-a-convenient-way-to-create-parcelable-data-classes-in-android-with-kotl)). Another option would be to keep the same way of passing the arguments, but make a helper class that returns a proper data class/map with all the attributes filled in.